.editorconfig

```diff
@@ -0,0 +1,23 @@
+root = true
+
+[*.{kt,kts}]
+ktlint_standard_no-blank-lines-in-chained-method-calls = disabled
```

Comment:
There is a couple of new rules I had to disable. I have fixed some of them in commits: https://github.com/getodk/collect/pull/5974/commits/9b7e6a47472f3f14ee76d33ac09fcbc72c0053b2 https://github.com/getodk/collect/pull/5974/commits/1ed546d4e2225b48925c017340943dbfb9dbabfd https://github.com/getodk/collect/pull/5974/commits/64c21076d055287a126f903b52fba9e52509fe82 https://github.com/getodk/collect/pull/5974/commits/9c57ee472ed2f140b53b872a20b40bda0be63567 https://github.com/getodk/collect/pull/5974/commits/0bd66c2657df2a0a9ae5f7f2c4d0b1c53c22d3c3
the others would either require updating a lot of classes or they do not make that much sense so it's something for separate pull requests.